airflow/utils.py

```diff
@@ -776,3 +778,33 @@ def logger(self):
             self._logger = logging.root.getChild(self.__class__.__module__ + '.' +self.__class__.__name__)
             return self._logger
 
+
+def get_sqla_engine(pool_size=5, pool_recycle=3600):
```

Comment:
Yeah, my concern is that if this setting value has issues, there's no easy way for users to fix it.